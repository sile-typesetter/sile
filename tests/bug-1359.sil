\begin[papersize=6in x 9in]{document}
\nofolios
\neverindent
\language[main=fr]
\font[size=50pt]

\font[size=10pt]{Lorem ipsum : « dolor » ; sit ? amet !}
\end{document}
