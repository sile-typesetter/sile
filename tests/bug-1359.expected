Set paper size 	432	648
Begin page
Mx 	21.6000
My 	39.9684
Set font 	Gentium Plus;10;400;;normal;;LTR
T	47 82 85 72 80 w=26.4014	(Lorem)
Mx 	50.6706
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	78.5399
T	29 w=2.2900	(:)
Mx 	83.4992
T	169 w=4.8535	(«)
Mx 	90.4747
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	114.5255
T	170 w=4.8535	(»)
Mx 	120.7003
T	30 w=2.2900	(;)
Mx 	125.6595
T	86 76 87 w=10.0146	(sit)
Mx 	136.9955
T	34 w=4.3311	(?)
Mx 	143.9957
T	68 80 72 87 w=20.6836	(amet)
Mx 	166.0006
T	4 w=2.7197	(!)
End page
Finish
